### DISCUSSION TITLE
建议更新英文模式

### DISCUSSION THREAD

**Discussion by tou002** [Meloong-Git/PCL#1389] [提问] [answered]
有个法国人问我我的启动器是啥我给他看根本看不懂

**Comment by WForst-Breeze** [comment]
PCL2 不会支持多语言，很抱歉🙁请见 #119

**Comment by zkitefly** [comment]
https://github.com/Hex-Dragon/PCL2/pull/119#issuecomment-1207343999

> 我暂时还不打算支持多语言……它不仅仅需要翻译的支持，而主要是英文的文本长度比中文更长，这导致许多界面需要重新进行布局，此外类似下载源的国际适配、第三方网站的多语言适配之类的都是很头秃的问题……
> 并且这会明显拖慢制作新功能的速度，现在已经够慢了 Orz

**Comment by Big-Cake-jpg** [comment, marked as answer]
如您要提交新功能提案，请前往 Issues 页面提交反馈。

与 #198 重复，感谢你的反馈。

下一次提交反馈前，请先记得使用搜索框在 Issues 列表内搜索是否有重复的 Issue，感谢理解，多谢配合。
